lib/src/models/chat_completion.dart

```diff
@@ -73,8 +73,9 @@ class ChatMessage {
 
   final String role;
   final String content;
+  @JsonKey(includeIfNull: false)
```

Comment:
Wouldn't be better to add it to `@JsonSerializable(includeIfNull: false)` instead of adding keys? Wdyt?